base/special/trig.jl

```diff
@@ -1073,6 +1073,26 @@ end
 
 tand(x::Real) = sind(x) / cosd(x)
 
+"""
+    sincosd(x)
+
+Simultaneously compute the sine and cosine of `x`, where `x` is in degrees.
+
+"""
+@inline function sincosd(x::Real)
```

Comment:
I don't think this needs to be declared inline.

Reply:
OK! Tomorrow morning as soon as I get access to my computer, I will update the PR.

Reply:
Hi @JeffBezanson !

I removed the `@inline`. Indeed, I was no decrease in performance.